### ISSUE TITLE
Copy of 1010D investigate the flow for remarriage

### ISSUE THREAD

**Issue by jamiefiore** [department-of-veterans-affairs/va.gov-team#90638]
[CAIA wants us to double check](https://dsva.slack.com/archives/C068Q9DQACX/p1722013531509259) that we are asking the right questions and supporting documents according the SME. 
Revisit the [flow in the Figma,](https://www.figma.com/design/UmAtr3ULQEInMXfNFwP0g0/Application-for-CHAMPVA?node-id=2017-51097&t=zUN6APrMa7plGviR-0) the [stakeholder notes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/champva/1010D/1010d-program-questions.md) and [the supporting dos matrix](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1713985425255/14109ce0d93bb1b1df050fea11b6a7fa029595f6?sender=u10700c583aaa99f809c88675).

- [ ] Revise any UI screens that are needed. 
- [ ] Notate any decisions on the [design decision page in the Figma file](https://www.figma.com/design/UmAtr3ULQEInMXfNFwP0g0/Application-for-CHAMPVA?node-id=1962-75450&t=zUN6APrMa7plGviR-0).

**Comment by jamiefiore** [comment]
@baltes @michaelclement Lois just approved this. Here's the [link to the marriage flow/content if the sponsor died](https://www.figma.com/design/UmAtr3ULQEInMXfNFwP0g0/Application-for-CHAMPVA?node-id=2097-7995&t=uh7KwhdyRYkUX7nI-4) and here is the [link to the flow if the sponsor is alive](https://www.figma.com/design/UmAtr3ULQEInMXfNFwP0g0/Application-for-CHAMPVA?node-id=2097-9174&t=uh7KwhdyRYkUX7nI-4). 

Basically what we are doing is updating the content in the upload screen and making it required on the sponsor died path and removing the marriage cert upload from the sponsor living path. 
Let me know if you have any questions.

**Comment by baltes** [comment]
Thanks @jamiefiore . @michaelclement I'll create a ticket for this.
